graphql/schema.js

```diff
@@ -47,6 +48,46 @@ type User {
   initials: String!
   email: String
   credentials: [Credential!]!
+  testimonial: Testimonial
+  facebookId: String
+  twitterHandle: String
```

Comment:
what are facebookId and twitterHandle for? Do they really need to be exposed via the API?

Reply:
They will be exposed on the profile page, if the user decides to expose them publicly. Would you prefer to wrap this data into some object like PublicSocialData or so?

WIP:
<img width="858" alt="screen shot 2017-10-16 at 09 53 40" src="https://user-images.githubusercontent.com/23520051/31600791-fd5b7386-b257-11e7-80cc-11fd7a433eb3.png">